src/main/java/soot/SootMethod.java

```diff
@@ -230,7 +230,7 @@ public boolean isPhantom() {
    * Returns true if this method is not phantom, abstract or native, i.e. this method can have a body.
    */
   public boolean isConcrete() {
-    return !isPhantom() && !isAbstract() && !isNative();
+    return !isPhantom() && !isAbstract();
```

Comment:
Sounds like a good idea, this implementation right here would brick things.

I wonder whether it would be better to do something like ```hasActiveBody() || ms != null```, so if it is **possible** to retrieve a body at all, which is what every caller seems to actually want to know.

Reply:
That would be an option, but it would also force us to refactor the Jimple Native reader to make it a proper ``MethodSource``.